README.md

```diff
@@ -67,16 +67,29 @@ Spotlight introduces functionality that depends on being able to send emails to
 
 See the [Spotlight wiki](https://github.com/sul-dlss/spotlight/wiki) for more detailed information on configuring Spotlight.
 
-## To start the development/test application
+# Developing Spotlight
 
- 1. Clone this repo, `cd` in and run `$ bundle install`
- 2. Start jetty and the dev server: `$ bundle exec rake spotlight:server` (this task will build a Spotlight-based application, start Solr, and run the built-in rails server)
- 3. Visit [http://localhost:3000](http://localhost:3000)
+Spotlight:
+
+* is a Rails engine and needs to be used in the context of a Rails application. We use [engine_cart](https://github.com/cbeer/engine_cart) to create an internal test application at spec/internal.
+* uses Solr as part of its integration tests. We use [jettywrapper](https://github.com/projecthydra/jettywrapper) to manage the Solr instance used for development and test.
+
+Our `$ rake ci` and `$ rake spotlight:server` tasks utilize Solr and the testing rails app automatically.
+
+##  More Information for Developers
+
+* [Contributing to Spotlight](https://github.com/sul-dlss/spotlight/wiki/Contributing-to-Spotlight)
+* [Testing](https://github.com/sul-dlss/spotlight/wiki/Testing)
 
 ## Tests
 
-Run tests:
+### Prerequisites
+
+PhantomJS (https://github.com/teampoltergeist/poltergeist#installing-phantomjs) is an addition requirement for testing javascript.
+
+### Run all the tests:
 
 ```
-$ rake
+$ rake ci
```

Comment:
yes, but I felt it was better to be explicit about the task.  It seemed like better documentation, somehow.  Otherwise there's a "the default rake task is to run all the tests" assumption baked in.